runtime/metricsview/astexpr.go

```diff
@@ -210,7 +224,12 @@ func (b *sqlExprBuilder) writeBinaryCondition(exprs []*Expression, op Operator)
 
 		// If not unnested, write the expression as-is
 		if !unnest {
-			return b.writeBinaryConditionInner(nil, right, leftExpr, op)
+			if lm, ok := b.lookupMeta[left.Name]; ok {
```

Comment:
Inside `b.sqlForName` above, it calls `b.ast.lookupDimension`, which returns the dimension object that already has the lookup info. 

So is it possible to avoid the `lookupMeta` member and either:
1. compute the lookup expression inside `sqlForName`, or
2. return the lookup info from `sqlForName`

I think that would also mirror how unnest is handled (i.e. returned from `sqlForName` instead of being a meta object on the `ast`).

Reply:
sure that can be done, I did this so lookup info is looked up only once instead of multiple times but looking at it now again it seems not necessary